packages/rocketchat-grant/server/index.js

```diff
@@ -12,9 +13,30 @@ import Providers, { middleware as providers } from './providers';
 import Settings from './settings';
 
 let grant;
+let storeAddress = process.env.SESSIONS_URL || 'mongodb://localhost:27017/rocketsession';
+
+if (storeAddress.indexOf('connectTimeoutMS=') < 0) {
+	if (storeAddress.indexOf('?') < 0) {
+		storeAddress += '?connectTimeoutMS=1200';
+	} else {
+		storeAddress += '&connectTimeoutMS=1200';
+	}
+}
+const store = new SessionStorage({
+	collection: process.env.SESSION_COLLECTION || 'mySessions',
```

Comment:
the default collection should follow our naming schema, and I think it should reflect it's being used by `grant` module, so I would say: `rocketchat_grant_sessions`

Reply:
done